nautobot_ssot/integrations/dna_center/diffsync/adapters/nautobot.py

```diff
@@ -89,9 +86,7 @@ def __init__(
     def load_regions(self):
         """Load Region data from Nautobt into DiffSync models."""
         try:
-            locations = OrmLocation.objects.filter(location_type=self.locationtype_map["Region"]).select_related(
-                "parent"
-            )
+            locations = OrmLocation.objects.filter(location_type=self.job.area_loctype).select_related("parent")
             for region in locations:
```

Comment:
Does `region` need to be changed to `area` throughout? Same question for `site` -> `building`

Reply:
If we really want to be exact, sure.

Reply:
These have been updated as requested.